docs/Basics/download-project.md

```diff
@@ -5,13 +5,11 @@
 If you ever need to save the contents of your project without pushing to a Git repository,
 you can download a zip file of the current project directly from the Studio settings menu.
 
-![Download a Project](../../assets/images/download-project-as-zip.gif)
+![](../../assets/images/download-project-as-zip.gif)
 
 To download a project:
 
 1. If you haven't already, select the project you would like to download from the Studio landing screen
 2. Navigate to the settings menu using the cog wheel at the left bottom corner of the screen
 3. Select the **Project Settings** option from the menu
 4. Click the **Download Project** button, which will start the download
-
```

Comment:
I felt like this was old enough it wasn't super interesting anymore as we probably don't have many v1.1.0 users at this point.